results-processor/main.py

```diff
@@ -204,8 +204,9 @@ def task_handler():
 
     ds = datastore.Client()
     secret = ds.get(ds.key('Uploader', '_processor'))['Password']
-    wptreport.create_test_run(report, labels, uploader, secret,
-                              results_gcs_path, raw_results_gcs_path)
+    id = wptreport.create_test_run(report, labels, uploader, secret,
+                                   results_gcs_path, raw_results_gcs_path)
+    assert (isinstance(id, int) and id != 0)
```

Comment:
This will fail... There's a high chance that the test run ID is of type `long`, not `int`, because of its range. And `long` isn't a subclass of `int`. In fact, they don't even have a common base class. (Well, they have an *abstract* common base class, `numbers.Integral`.)

I'd say just `assert id` (which makes sure it's not `None` or 0) and don't worry about the type, the Pythonic way :)